__tests__/toc.test.ts

```diff
@@ -0,0 +1,37 @@
+import {execSync} from 'child_process';
```

Comment:
let's make the table of contents generation an option that is enabled by default, but it can be disabled by passing a flag

we won't write a separate test, but we'll add a snapshot with the generated table of contents to the current test 
https://github.com/diplodoc-platform/pdf-generator/blob/master/__tests__/index.test.ts